checkST2110.js

```diff
@@ -112,33 +113,35 @@ const test_10_74_1 = (sdp, params) => {
       zeroCheck = zeroCheck.map(z => +(z.trim().split('=')[2]));
       for (let x = 0; x < zeroCheck.length; x++) {
         if (zeroCheck[x] !== 0) {
-          errors.push(new Error(`Stream ${s + 1}: The 'mediaclk' attribute shall have a zero offset when direct-referenced PTP timing is in use, as per ST 2110-10 Section 7.4.`));
+          errors.push(new Error(`Stream ${s + 1}: The 'mediaclk' attribute shall have a zero offset when direct-referenced PTP timing is in use, as per ST 2110-10:2017 Section 7.4 and ST 2110-10:2022 Section 8.3.`));
         }
       }
     }
   }
   if (params.verbose && errors.length == 0) {
-    console.log('Test Passed: ST 2110-10 Section 7.4 Test 1 - Where mediaclk:direct is used with PTP, offset value is zero');
+    console.log('Test Passed: ST 2110-10 Section 7.4 and ST 2110-10:2022 Section 8.3 Test 1 - Where mediaclk:direct is used with PTP, offset value is zero');
   }
   return errors;
 };
 
-// ST 2110-10 Section 8.1 Test 1 - Shell have media-level mediaclk per stream
+// ST 2110-10:2017 Section 8.1
+// ST 2110-10:2022 Section 8.3 Test 1 - Shell have media - level mediaclk per stream
```

Comment:
Let's fix the typo `Shell` to `Shall` and remove spaces around hyphen in `media-level` as before

Reply:
I think since the name of the test function still follows the 2017 spec, better to move the "Test 1..." description to the first line?